src/components/Hero/Heading/Button/ModalWindow.jsx

```diff
@@ -1,8 +1,12 @@
 import { Dialog } from '@headlessui/react'
 import React from 'react'
 import closeImg from '../../../../img/closeButton.svg'
+import closedEyeImg from '../../../../img/eye-password-hide.png'
+import eyeImg from '../../../../img/eye.png'
 import styles from './ModalRegistrationForm.module.scss'
 const ModalWindow = ({ isModalOpen, setIsModalOpen }) => {
+  const [showPassword, setShowPassword] = React.useState(false) // useState for password
+  const [isShowed, setIsShowed] = React.useState(false)
   return (
```

Comment:
Add blank line before component & return

`import styles from './ModalRegistrationForm.module.scss'

const ModalWindow = ({ isModalOpen, setIsModalOpen }) => {
  const [showPassword, setShowPassword] = React.useState(false) // useState for password
  const [isShowed, setIsShowed] = React.useState(false)

  return (`
  
  It will be more readable